### DISCUSSION TITLE
PnP -Search Filters web part French translation is wrong for Date range template

### DISCUSSION THREAD

**Discussion by rpochepovich** [microsoft-search/pnp-modern-search#3381] [Q&A] [answered]
In my English/French intranet site the PnP -Search Filters web part hosted on French page shows wrong translations for the Date range.
The From-To labels are translated as De-À. It however should be translated as Du-Au.
"De" and "À" would be how one addresses an email or a letter. 
Which in English would be "from" and "to" as well, but the translation is different in French depending on the context.
My understanding is an automated translation is used, which without context, is not able to differentiate.
My question is: is there a way to replace wrong translations?

**Comment by kasperbolarsen** [comment, marked as answer]
Absolutly. I have updated the French language file and the PR is awaiting approval. The new value will most likely be available in the next release.

**Reply by rpochepovich** [reply]
This is amazing! Thank you so much! I did not expect such a prompt reaction. This is why I did not list all the corrections, and there is 1 more. Here is a full list of corrections that as far as my team responsible for French content found:

Web part                   |  Section	                        | Existing French      |    Correct French
-----------------------|-------------------------------|---------------------|------------------
PnP-Search Filters	  |  Date Range labels	        | De	                        |   Du
PnP-Search Filters	  |  Date Range labels	        | À	                        |   Au
PnP-Search Results	  |  Number of results on top  |  NN résultats de	|   NN résultats pour

Also a question: when the next release is planned? Rough estimation to give to my business users on the timelines of when we can deploy the corrected version.

**Comment by kasperbolarsen** [comment]
Please review https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/src/loc/fr-fr.js and let us know of any issues , batching the changes is preferred :-)

**Reply by rpochepovich** [reply]
Hello, Kasper,
I have asked our French content team to review, but right now they are too busy with other staff and only might put it into their schedule after the New Year.
At this point I think we can go ahead with those 3 changes. It's possible that all other translations are fine and no sense to wait.

**Reply by kasperbolarsen** [reply]
No worries,  We'll wait as no new release is planned anytime soon.

**Reply by rpochepovich** [reply]
When is the new release planned? Roughly

**Reply by rpochepovich** [reply]
Also, is there a way for me to fix it locally? Not to wait until the new release?

**Reply by kasperbolarsen** [reply]
You can absolutely correct the errors in the translation file and build the project your self , see https://microsoft-search.github.io/pnp-modern-search/how-to-contribute/

**Reply by rpochepovich** [reply]
Thank you, will try

**Reply by kasperbolarsen** [reply]
> When is the new release planned? Roughly

Hard to say. Once the bugs in the current dev branch has been resolved, which again depends on someone have too much free time on their hands 😁

**Reply by rpochepovich** [reply]
Makes sense. Thank you for your help!

**Comment by sabineFPSFA** [comment]
@rpochepovich: Please note there is another typo in the French translation of the date range: "derniière" should be "dernières" (cfr enclosed image)
![PnP search refiner French typo](https://github.com/user-attachments/assets/c3d5a110-5b07-4928-b73b-5dc120e67a93)

**Reply by kasperbolarsen** [reply]
so like this?
![image](https://github.com/user-attachments/assets/07f4837c-e00c-4642-8f20-81b72550c58f)

**Reply by sabineFPSFA** [reply]
Perfect! :)
